src/xarray_regrid/methods/conservative.py

```diff
@@ -98,7 +100,11 @@ def conservative_regrid_dataset(
                 da = dataarrays[i].transpose(coord, ...)
                 dataarrays[i] = apply_weights(da, weights, coord, target_coords)
 
-    return xr.merge(dataarrays)  # TODO: add other coordinates/data variables back in.
+    for da, _attr in zip(dataarrays, da_attrs, strict=True):
+        da.attrs = _attr
```

Comment:
Add the coordinates' attributes here:

```suggestion
    for da, _attr in zip(dataarrays, da_attrs, strict=True):
        da.attrs = _attr
    for dcrd, _attr in zip(datacoords, dcrds_attrs, strict=True):
        dcrd.attrs = _attr
```